woocommerce/wc-admin
# update chart type query string key to chartType

Fixes #1450

Replace the chart `type` query string with `chartType` to avoid conflicts with stock report `type` query string.

### Accessibility




### Screenshots

### Detailed test instructions:

- Open the dashboard
- load the product report
- switch to bar graph and check URL has chartType=bar
- switch back to line and check URL has chartType=line



If I should have completed accessibility tests for this, please let me know which ones.
